packages/dev/docs/src/HeaderInfo.js

```diff
@@ -46,12 +47,14 @@ export function HeaderInfo(props) {
           </tr>
         </tbody>
       </table>
-      <div className={styles['resourceCardGroup']}>
-        {sourceData.map((source) => (
-          <ResourceCard type={source.type} url={source.url} />
-        ))}
-        <ResourceCard type="GitHub" url={`https://github.com/adobe-private/react-spectrum-v3/tree/master/packages/${encodeURI(packageData.name)}`} />
-        <ResourceCard type="NPM" url={`https://www.npmjs.com/package/${encodeURI(packageData.name)}`} />
+      <div className={classNames(styles['flex-container'])}>
```

Comment:
Why not use the <Flex> component here? I think it should server render ok...

Reply:
ah, it slipped my mind that i could, i'll do that